tests/test_objective_funs.py

```diff
@@ -319,19 +311,19 @@ def test(eq):
         # precise_QA should have lower QA than QH
         obj = QuasisymmetryTwoTerm(eq=eq1, helicity=helicity_QA)
         obj.build()
-        f1 = obj.compute_scalar(*obj.xs(eq1))
+        f1 = obj.compute_scalar(*obj.xs(eq1), constants=obj.constants)
```

Comment:
In theory either is fine. Passing in constants should in theory lead to faster compiles since it doesn't try to bake the constants into the compiled code, though in practice who knows.